analyzers/tests/SonarAnalyzer.Test/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -223,32 +223,32 @@ public void Verify_UnchangedFiles(string unchangedFileName, bool expectedProtobu
-                        VerifyReferences(orderedSymbols[1].Reference, 9, 41);               // x
-                        VerifyReferences(orderedSymbols[1].Reference, 9, 42);               // y
-                        VerifyReferences(orderedSymbols[1].Reference, 9, 44, 41);           // LocalMethod
-
-                        VerifyReferencesColumns(orderedSymbols[1].Reference, 13, 4, 4, 19, 31);   // currentCount: line 4
-                        VerifyReferencesColumns(orderedSymbols[1].Reference, 13, 6, 6, 1, 13);    // currentCount: line 6
-                        VerifyReferencesColumns(orderedSymbols[1].Reference, 13, 20, 20, 8, 20);  // currentCount: line 20
-                        VerifyReferencesColumns(orderedSymbols[1].Reference, 13, 48, 48, 26, 38); // currentCount: line 48
-
-                        orderedSymbols[3].FilePath.Should().EndWith("RazorComponent.razor"); // RazorComponent.razor
+                        orderedSymbols.Select(x => Path.GetFileName(x.FilePath)).Should().BeEquivalentTo("Razor.razor", "Razor.razor.cs", "RazorComponent.razor", "ToDo.cs");
+                        orderedSymbols[0].FilePath.Should().EndWith("Razor.razor");
+
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 14, 5, 7, 21, 49);        // currentCount
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 17, 11, 21, 22);          // IncrementAmount
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 19, 9, 53);               // IncrementCount
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 35, 35);                  // x
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 38, 29, 35);              // todos
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 40, 26);                  // AddTodo
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 42);                      // x
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 43);                      // y
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 45, 42);                  // LocalMethod
+                        VerifyReferences(orderedSymbols[0].Reference, 10, 58, 52);                  // AdditionalAttributes
+
+                        VerifyReferencesColumns(orderedSymbols[0].Reference, 14, 5, 5, 19, 31);     // currentCount: line 5
+                        VerifyReferencesColumns(orderedSymbols[0].Reference, 14, 7, 7, 1, 13);      // currentCount: line 7
+                        VerifyReferencesColumns(orderedSymbols[0].Reference, 14, 21, 21, 8, 20);    // currentCount: line 21
+                        VerifyReferencesColumns(orderedSymbols[0].Reference, 14, 49, 49, 26, 38);   // currentCount: line 49
+
+                        orderedSymbols[1].FilePath.Should().EndWith("RazorComponent.razor");        // RazorComponent.razor
```

Comment:
I wonder why the order has changed between `RazorComponent.razor` and `Razor.razor.cs`

Reply:
Indeed surprising.